src/main/java/ch/njol/skript/expressions/ExprCustomModelData.java

```diff
@@ -0,0 +1,106 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Custom Model Data")
+@Description("Get/set the CustomModelData tag for an item. (Value is an integer between 0 and 99999999)")
+@Examples({"set custom model data of player's tool to 3",
+	"set {_model} to custom model data of player's tool"})
+@RequiredPlugins("1.14+")
+@Since("INSERT VERSION")
+public class ExprCustomModelData extends SimplePropertyExpression<ItemStack, Long> {
+	
+	static {
+		if (Skript.methodExists(ItemMeta.class, "hasCustomModelData")) {
+			register(ExprCustomModelData.class, Long.class, "[custom] model data", "itemstack");
```

Comment:
Same thing here, also, why does it not take multiple items?

Reply:
fixed, and now takes multiple items